app/src/main/java/org/lineageos/glimpse/repository/MediaFlow.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The LineageOS Project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.lineageos.glimpse.repository
+
+import android.content.Context
+import android.database.Cursor
+import android.os.Build
+import android.provider.MediaStore
+import androidx.core.os.bundleOf
+import androidx.loader.content.GlimpseCursorLoader
+import androidx.loader.content.Loader.OnLoadCompleteListener
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+import org.lineageos.glimpse.models.Media
+import org.lineageos.glimpse.query.*
+import org.lineageos.glimpse.utils.MediaStoreBuckets
+import org.lineageos.glimpse.utils.MediaStoreRequests
+
+class MediaFlow(private val context: Context, private val bucketId: Int?) {
+    fun flow() = callbackFlow {
```

Comment:
no, add back the type, callbackFlow by default is a mutableFlow, we want a Flow
ditto for albums